<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Date/日期</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_city">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>City/城市</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_aft">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>117</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>后天</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_tod">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>117</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>今天</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_tom">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>117</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>明天</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>signal1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>signal1()</slot>
 </slots>
</ui>
